Display all contributions in 'My contributions' (#5575)
indico/indico: CHANGES.rst

^^^^^^^^^^^^

- Display program codes in 'My contributions' (:pr:`5573`)

> - Display all contributions in 'My contributions' categorized by author/speaker/submitter (:pr:`5575`)

Reviewer: This line is super long...

```suggestion
- Display all contributions in 'My contributions' and not just those with
  submitter privileges (:pr:`5575`)
```

How about this instead?
